src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -632,14 +719,22 @@ export default function (AB) {
        *
        * @param {string} name  the menuName() of the submenu that was selected.
        */
-      onChange(name) {
+
+      onClick(name) {
+         // show Field Type popup
+         $$(this.ids.chooseFieldType).hide();
+         $$(this.ids.fieldSetting).show();
+
+         // set title name by each field type
+         $$(this.ids.title).setValue(L(`<span>Create Field: ${name}</span>`));
```

Comment:
We can translate the first part but the "name" will have its translation stored elsewhere so try this...

`$$(this.ids.title).setValue(L("Create Field:") + " " + name");`